APTITUDE/APTITUDETEST.Data/Data/AdminAuthenticationRepository.cs

```diff
@@ -0,0 +1,307 @@
+﻿using AptitudeTest.Common.Helpers;
+using AptitudeTest.Core.Entities.Admin;
+using AptitudeTest.Core.Interfaces;
+using AptitudeTest.Core.ViewModels;
+using AptitudeTest.Data.Common;
+using APTITUDETEST.Common.Data;
+using APTITUDETEST.Core.Entities.Users;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System.Net.Mail;
+using System.Text;
+
+namespace AptitudeTest.Data.Data
+{
+    public class AdminAuthenticationRepository : IAdminAuthenticationRepository
+    {
+        #region Properies
+        private readonly AppDbContext _context;
+        static IConfiguration _appSettingConfiguration;
+        public static Dictionary<string, TokenVm> RefreshTokens = new Dictionary<string, TokenVm>();
+        #endregion
+
+        #region Constructor
+        public AdminAuthenticationRepository(AppDbContext context, IConfiguration appSettingConfiguration)
+        {
+            _context = context;
+            _appSettingConfiguration = appSettingConfiguration;
+        }
+        #endregion
+
+        #region Methods
+
+        #region Login
+        public async Task<JsonResult> Login(LoginVm loginVm)
+        {
+            try
+            {
+                if (!string.IsNullOrEmpty(loginVm.Email) && !string.IsNullOrEmpty(loginVm.Email))
```

Comment:
Invert this condition and reduce nested conditional blocks (remove else blocks)

Reply:
changes done